front/app/components/FilterBoxes/InputFilterCollapsible.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import {
+  Box,
+  CollapsibleContainer,
+  colors,
+} from '@citizenlab/cl2-component-library';
+
+interface Props {
+  title: string;
+  children: JSX.Element;
```

Comment:
```suggestion
    children: JSX.Element | JSX.Element[];
```
So we don't need add an extra wrapper when using this component.